ASmouzh/Htask_v2_01/list_of_commamds

```diff
@@ -0,0 +1,143 @@
+ 133  cd gitTest
```

Comment:
Please use github markdown to formatting your reports: https://guides.github.com/features/mastering-markdown/